wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -0,0 +1,96 @@
+                        {% trans "Refresh" %}
+                    {% endfragment %}
+                    {% dialog_toggle classname="button button-small button-secondary chooser__choose-button" text=refresh_button_content dialog_id="w-unsaved-changes-dialog" %}
+                </div>
+            {% enddropdown %}
+        {% elif session.is_editing %}
+            {% fragment as avatar %}
+                {% avatar user=session.user classname="w-mx-[1px] w-w-7 w-h-7 w-border-2 w-border-warning-100" %}
+            {% endfragment %}
+
+            {% blocktranslate trimmed with user_name=session.user|user_display_name|default:_("system") asvar has_unsaved_changes_message %}
+                {{ user_name }} has unsaved changes
+            {% endblocktranslate %}
+
+            {% dropdown theme="popup" toggle_label=avatar toggle_classname="w-p-0" %}
+                <div class="w-label-2 w-flex w-items-center w-gap-1.5 w-p-1">
+                    {% icon name="warning" classname="w-text-warning-100 w-w-5 w-h-5" %}
+                    {{ has_unsaved_changes_message|capfirst }}
+                </div>
+            {% enddropdown %}
+        {% else %}
+            {% fragment as avatar %}
+                {% avatar user=session.user classname="w-mx-[1px] w-w-7 w-h-7 w-border-2 w-border-surface-field hover:w-border-positive-100" %}
+            {% endfragment %}
+
+            <div>
+                <button class="w-p-0 w-bg-transparent" type="button" data-controller="w-tooltip">
+                    {{ avatar }}
+                    <div class="w-flex w-flex-col w-items-center w-justify-center" data-w-tooltip-target="content" hidden>
+                        <span class="w-font-semibold">{{ session.user|user_display_name|default:_("system")|capfirst }}</span>
+                        <span>{% trans "Currently viewing" %}</span>
+                    </div>
+                </button>
+            </div>
+        {% endif %}
+    {% endfor %}
+    {% if sessions|length > 4 %}
+        {% fragment as more_sessions_toggle %}
+            <div class="w-flex w-items-center w-justify-center w-bg-surface-page w-rounded-full w-w-7 w-h-7 w-border-2 w-border-border-furniture hover:w-border-positive-100 w-font-bold w-text-11">
+                +{{ sessions|length|add:"-4" }}
+            </div>
+        {% endfragment %}
+
+        {% dropdown theme="drilldown" toggle_label=more_sessions_toggle toggle_classname="w-p-0 w-mx-[1px]" %}
+            <div class="w-flex w-flex-col w-gap-4 w-p-2">
+                {% for session in sessions|slice:"4:" %}
+                    <div class="w-flex w-items-center">
+                        {% avatar user=session.user size="small" %}
+                        <span class="w-text-14">{{ session.user|user_display_name|default:_("system")|capfirst }}</span>
+                    </div>
+                {% endfor %}
+            </div>
+        {% enddropdown %}
+    {% endif %}
+</div>
+
+{% if sessions.0.revision_id %}
```

Comment:
This is assuming the user who has the newest `revision_id` is always the first user in the list, but it's OK because the backend does the ordering logic.